src/views/Dashboards/Dashboards.js

```diff
@@ -117,7 +119,8 @@ TabPanel.propTypes = {
   src: PropTypes.string.isRequired,
   title: PropTypes.string.isRequired,
   scenarioName: PropTypes.string,
-  scenarioId: PropTypes.string
+  scenarioId: PropTypes.string,
+  noScenario: PropTypes.array.isRequired
```

Comment:
... whereas it seems to be defined as a required array here.

Reply:
Could you please fix the console error "Warning: Failed prop type: Invalid prop `noScenario` of type `boolean` supplied to `TabPanel`, expected `array`." ?

Reply:
Done